README.md

```diff
@@ -85,6 +85,35 @@ end
 In this case, Rails will look for a serializer named `PostSerializer`, and if
 it exists, use it to serialize the `Post`.
 
+### Built in Adapters
+
+There are two built in adapters - `:json` and `:json_api`. You can choose which you'd
+like to use with the `config.adapter` setting, as above:
+
+```ruby
+ActiveModel::Serializer.config.adapter = :json_api
+```
+
+The main difference between the default `:json` and `:json_api` adapters is that 
+`:json_api` includes a "root" element (usually the class name of the serialized 
+object). This is roughly equivalent to Rails' `include_root_in_json` setting and
+useful for compatibility with frameworks such as Ember.
+
```

Comment:
I think this paragraph is not needed. A simple link to http://jsonapi.org would avoid confusion because the differences are enormous.